package.json

```diff
@@ -11,7 +12,7 @@
   "scripts": {
     "build": "npm run build:cjs && npm run build:esm",
     "build:cjs": "babel src --out-dir lib/cjs",
-    "build:esm": "BABEL_ENV=es6 babel src --out-dir lib/esm",
+    "build:esm": "BABEL_ENV=es6 babel src --out-dir lib/esm && echo '{}' > lib/esm/package.json",
```

Comment:
The empty package definition in `lib/esm/package.json` overrides the top-level `type: 'commonjs'` without requiring strict ESM module resolution ie. fully qualified `import` paths.